content/how-to/split-code.md

```diff
@@ -1,11 +1,30 @@
 ---
 title: How to Split Code?
+contributors:
+  - pksjce
 ---
-> what is code splitting
-> System.import
-> dynamic System.import
-> require.ensure
-> AMD require
-
-> see also [[Output]]
-> see also [[dynamic dependencies]]
+
+Code splitting is the most compelling feature for  `webpack` usage. `webpack`, with some configuration, can tweak your app such that, you can split your code into various bundles and load them on demand.
```

Comment:
Can you take another lok at the second sentence? It has too many comma's in it, making it a bit hard to read.